Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -500,16 +500,23 @@ - (void)update
     [self refreshSortBarText];
 
     BOOL isTrashOnScreen = self.isDeletedFilterActive;
-    BOOL isNotEmpty = !self.isListEmpty;
 
-    self.emptyTrashButton.enabled = isTrashOnScreen && isNotEmpty;
+    [self setEmptyTrashEnabled];
     self.tableView.allowsSelection = !isTrashOnScreen;
     
     [self displayPlaceholdersIfNeeded];
     [self updateNavigationBar];
     [self hideRatingViewIfNeeded];
 }
 
+- (void)setEmptyTrashEnabled
+{
+    BOOL isTrashOnScreen = self.isDeletedFilterActive;
+    BOOL isNotEmpty = !self.isListEmpty;
+
+    self.emptyTrashButton.enabled = isTrashOnScreen && isNotEmpty;
+}
```

Comment:
Two notes!

1. Can we rename this method to something like `refreshEmptyTrashState` or `refershEmptyTrashButton`?
   Why: by convention, in ObjC, properties would "hide" access to methods named `setProperty:value` or `property` (getter). Although this API does update the trash button, it doesn't accept values.

2. We're moving all new code over to Swift. Mind placing this one in `SPNoteListViewController+Extensions`?